### PR TITLE
Audit log accordions

### LINKED ISSUE #1341: Audit Log Date Accordians
### Description

In accordance with the [new design](https://github.com/flowforge/flowforge/issues/428#issuecomment-1271674418) shown in #428, the audit logs should be collapsable on a date-by-date basis to make navigation easier.

### Epic/Story

#428

### PR DESCRIPTION
**Pull request by joepavitt**
Closes #1341 

Accordions disabled in Project Overview, enabled elsewhere.

https://user-images.githubusercontent.com/99246719/207592656-a8589ce3-5d81-423c-a168-886b0cda108c.mov

### CODE REVIEW

**Comment by Pezmc** [comment]
I think the _animation_ on the project overview should also be disabled, as it is right now the page grows in size vertically every time you navigate to it.

File: test/e2e/frontend/cypress/tests/team/audit-log.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+describe('FlowForge - Team Audit Log', () => {
+    // let team // , project
```
Comment (Pezmc) [inline test/e2e/frontend/cypress/tests/team/audit-log.spec.js:2]: ✂️

File: test/e2e/frontend/cypress/tests/team/audit-log.spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            log: [],
+            meta: {}
+        }).as('getAuditLog')
+
+        cy.visit('team/ateam/audit-log')
+        cy.wait(['@getAuditLog'])
+
+        cy.get('.ff-no-data').should('exist')
+    })
+})
```
Comment (Pezmc) [inline test/e2e/frontend/cypress/tests/team/audit-log.spec.js:20]: Are you adding test coverage of the accordion behaving? Or is this elsewhere?

**Review by Pezmc — APPROVED** [review]
I had one UX comment, everything else is non blocking and this looks great!

File: test/e2e/frontend/cypress/tests/team/audit-log.spec.js (hunk 2 above)
Comment (joepavitt) [reply test/e2e/frontend/cypress/tests/team/audit-log.spec.js:20]: good point - will put it in

File: test/e2e/frontend/cypress/tests/team/audit-log.spec.js (hunk 1 above)
Comment (joepavitt) [reply test/e2e/frontend/cypress/tests/team/audit-log.spec.js:2]: legacy - but will remove it here anyway

**Pull request merged**